src/resolver.rs

```diff
@@ -2435,7 +2447,8 @@ impl FailForVet {
         &self,
         out: &mut dyn Write,
         report: &ResolveReport,
-        cfg: &Config,
+        _cfg: &Config,
```

Comment:
Perhaps we should drop this argument now?

Reply:
I want to keep some of these spiderings around in case we ever have more formatting flags we want to dump into here.